src/gui/folder.cpp

```diff
@@ -641,8 +569,12 @@ void Folder::startVfs()
         _vfs->fileStatusChanged(stateDbFile + QStringLiteral("-wal"), SyncFileStatus::StatusExcluded);
         _vfs->fileStatusChanged(stateDbFile + QStringLiteral("-shm"), SyncFileStatus::StatusExcluded);
         _engine->setSyncOptions(loadSyncOptions());
+        registerFolderWatcher();
         _vfsIsReady = true;
-        slotScheduleThisFolder();
+        if (canSync()) {
```

Comment:
I think a comment that explains why we can safely ignore the "cannot sync" condition here would help readers understand the code better.

Reply:
Please be constructive and suggest something, documenting the function canSync here is definitely the wrong place.